WordPress/gutenberg
# Archive Title Block: Refactor settings panel to use ToolsPanel

## What?
Part of: https://github.com/WordPress/gutenberg/issues/67813

## Why?
Archive Title Block code to include ToolsPanel instead of PanelBody.

## How?
Refactored Archive Title Block code to include ToolsPanel instead of PanelBody.

## Testing Instructions

- Login to Dashboard
- Edit any post/ page
- Add Archive Title block.
- Check settings panel of the Archive title block and verify the change.

## Screenshots or screencast 

**Before**
![before_1](https://github.com/user-attachments/assets/bf30da6d-92c2-46dc-aa54-a6e07b075e50)


**After**
![after_1](https://github.com/user-attachments/assets/67826dc6-b335-4873-879f-125342cacea1)
